src/controllers/auth.controller.js

```diff
@@ -0,0 +1,141 @@
+import { User } from "../models/user.js";
+import { userService } from "../services/user.service.js";
+import { jwtService } from "../services/jwt.service.js";
+import { ApiError } from "../exeptions/api.error.js";
+import bcrypt from 'bcrypt';
+import { toeknService } from "../services/token.service.js";
```

Comment:
There is a typo in the import statement. It should be `tokenService` instead of `toeknService`.